app/screens/receive-bitcoin-screen/payment/helpers.ts

```diff
@@ -69,3 +71,15 @@ export const generateFutureLocalTime = (secondsToAdd: number): string => {
 
   return `${hours}:${minutes}${period}`
 }
+
+export const prToDateString = (paymentRequest: string, network: Network) => {
+  let dateString
+  try {
+    dateString = decodeInvoiceString(
+      paymentRequest,
+      network as NetworkLibGaloy,
+    ).timeExpireDateString
+    /* eslint-disable-next-line no-empty */
```

Comment:
We don't want to eat the error

At least show it in console.error
if there is nothing actionable on it